packages/kotti-ui/source/kotti-breadcrumb/KtBreadcrumb.vue

```diff
@@ -25,11 +25,34 @@
 	</div>
 </template>
 
-<script>
-export default {
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api'
+import { isBoolean, isString } from 'lodash'
+
+import { isYocoIcon } from '../validators'
+
+import { KottiBreadcrumb } from './types'
+
+const breadcrumbValidator = (
+	value: KottiBreadcrumb.Breadcrumb,
```

Comment:
This should be `unknown` or `any`, `x is y` functions are trying to assert that `y` is a sub-type of `x`, otherwise you could just return `true`